interface/src/app/utils.js

```diff
@@ -0,0 +1,41 @@
+angular.module('adage.utils', [
+
+])
+
+.factory('MathFuncts', [function() {
+  return {
+    /*
+     * The binomLog and hyperGeometricTest functions are based heavily on the
+     * 'binomlog' and 'hyperg' functions in /integrator/static/js/imp.js
+     * See code here:
+     * https://bitbucket.org/cgreene/integrator/src/1194f7ae1b11057a4649114e1f55570542acbcde/integrator/static/js/imp.js?at=giant&fileviewer=file-view-default
+    */
+    binomLog: function(n, k) {
+      var coeff = 0;
+      for (var i = n - k + 1; i <= n; i++) {
+        coeff += Math.log(i);
+      }
+      for (i = 1; i <= k; i++) {
+        coeff -= Math.log(i);
+      }
+      return coeff;
+    },
+
+    hyperGeometricTest: function(k, m, n, N) {
```

Comment:
I just added some more comments/documentation about how these variable names are common in online descriptions of the hypergeometric test, and also they are the variable names used by Aaron in the corresponding functions in imp.js. Do you think those look good?